docs/src/specification/encoding.rst

```diff
@@ -5,30 +5,42 @@ Most repo commands and status reports are Data packets whose Content contains
 ``RepoCommandParam`` or ``RepoCommandRes`` structure.
 These Data are issued via Pub-Sub protocol.
 Each ``RepoCommandParam`` and ``RepoCommandRes`` contains
-multiple ``ObjectParam`` and ``ObjectResult``, resp.
+multiple ``ObjParam`` and ``ObjStatus``, resp.
 These structures are defined as follows:
 
 .. code-block:: abnf
 
-    ObjectParam =
+    ObjParam =
         Name
         [ForwardingHint]
         [StartBlockId]
         [EndBlockId]
         [RegisterPrefix]
 
-    ObjectResult =
+    SyncParam =
+        SyncPrefix
+        [RegisterPrefix]
+        [DataNameDedupe]
+        [Reset]
+
+    ObjStatus =
         Name
         StatusCode
         [InsertNum]
         [DeleteNum]
 
+    SyncStatus =
```

Comment:
The use of SyncStatus shall be mentioned in **sync/join** and **sync/leave** pages.

Reply:
Well, actually it is not used. Though the server returns this struct, the client discards it.